todo/src/actions/index.js

```diff
@@ -0,0 +1,32 @@
+
+export const TO_DO_CREATE = 'TO_DO_CREATE';
+export const TO_DO_DESTROY = 'TO_DO_DESTROY';
+export const TO_DO_COMPLETE = 'TO_DO_COMPLETE';
+export const TO_DO_CLEAR = 'TO_DO_CLEAR';
+
+export function toDoCreate(toDoTask) {
```

Comment:
```suggestion
export const toDoCreate = (toDoTask) => ({
        type: TO_DO_CREATE,
        task: toDoTask
})
```

Reply:
You can extract a little boilerplate when using these es6 arrow functions with implicit returning.